### DISCUSSION TITLE
王小云通过这句话想表达什么？

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#107] [Q&A]
https://github.com/rdrg109/subtitles-npcr/blob/6775350da747b9371af7bc36852bdd3dae1f91f9/sentences/44-1.yaml#L121-L125

**Comment by rodrigo-morales-1** [comment]
This question is related to the meaning of a sentence, so it should be in "Discussions". I've already converted it into a discussion.

In this repository, we only create issues when there's a direct modification in any of the files.

**Comment by fhxshrs** [comment]
有些商家口中的“买一送一”并非“买一件商品，送等价的同一件商品”，而是“买一件商品，送一个低廉的小玩意”，所以王小云怀疑这位商家也是如此。

**Reply by rodrigo-morales-1** [reply]
谢谢你！明白了！我修改了英语翻译和西语翻译 f6a0c99 如果你不同意，请告诉我。
